Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -325,13 +325,13 @@ private void SetVisualState(VisualState state, bool animate)
 
         private void SetDetailsContent()
         {
-            if ((SelectedItem != null) && (_detailsPresenter != null))
+            if (_detailsPresenter != null)
             {
-                _detailsPresenter.Content = MapDetails == null
-                    ? SelectedItem
-                    : MapDetails(SelectedItem);
+               _detailsPresenter.Content = MapDetails == null
```

Comment:
No worries, I have different standards installed on this machine - Making sure spacing is consistent IS a battle worth fighting.